test/3x/torch/quantization/weight_only/test_awq.py

```diff
@@ -157,3 +157,73 @@ def test_quant_lm_head(self):
         assert (
             id(model.model.decoder.embed_tokens.weight) == lm_head_id
         ), "The tied lm_head weight is not deep copied, please check!"
+
+    def test_awq_absorb_to_layer(self):
+        absorb_to_layer = {
+            (
+                "transformer.h.0.attn.q_proj",
+                "transformer.h.0.attn.k_proj",
+                "transformer.h.0.attn.v_proj",
+                "transformer.h.0.mlp.fc_in",
+            ): "transformer.h.0.ln_1",
+            ("transformer.h.0.attn.out_proj",): "transformer.h.0.attn.out_proj",
+            ("transformer.h.0.mlp.fc_out",): "transformer.h.0.mlp.fc_out",
+            (
+                "transformer.h.1.attn.q_proj",
+                "transformer.h.1.attn.k_proj",
+                "transformer.h.1.attn.v_proj",
+                "transformer.h.1.mlp.fc_in",
+            ): "transformer.h.1.ln_1",
+            ("transformer.h.1.attn.out_proj",): "transformer.h.1.attn.out_proj",
+            ("transformer.h.1.mlp.fc_out",): "transformer.h.1.mlp.fc_out",
+            (
+                "transformer.h.2.attn.q_proj",
+                "transformer.h.2.attn.k_proj",
+                "transformer.h.2.attn.v_proj",
+                "transformer.h.2.mlp.fc_in",
+            ): "transformer.h.2.ln_1",
+            ("transformer.h.2.attn.out_proj",): "transformer.h.2.attn.out_proj",
+            ("transformer.h.2.mlp.fc_out",): "transformer.h.2.mlp.fc_out",
+            (
+                "transformer.h.3.attn.q_proj",
+                "transformer.h.3.attn.k_proj",
+                "transformer.h.3.attn.v_proj",
+                "transformer.h.3.mlp.fc_in",
+            ): "transformer.h.3.ln_1",
+            ("transformer.h.3.attn.out_proj",): "transformer.h.3.attn.out_proj",
+            ("transformer.h.3.mlp.fc_out",): "transformer.h.3.mlp.fc_out",
+            (
+                "transformer.h.4.attn.q_proj",
+                "transformer.h.4.attn.k_proj",
+                "transformer.h.4.attn.v_proj",
+                "transformer.h.4.mlp.fc_in",
+            ): "transformer.h.4.ln_1",
+            ("transformer.h.4.attn.out_proj",): "transformer.h.4.attn.out_proj",
+            ("transformer.h.4.mlp.fc_out",): "transformer.h.4.mlp.fc_out",
```

Comment:
After offline sync, we decide to use below format for ease-of-use.
`absorb_layer_dict = {
    "ln_1": (
        "attn.q_proj",
        "attn.k_proj",
        "attn.v_proj",
        "mlp.fc_in",
    ),
    "attn.v_proj": "attn.out_proj",
    "mlp.act": "mlp.fc_out",
}
`

Reply:
https://github.com/intel/neural-compressor/pull/1920/commits/cc61136b8d42a853339f854e3164a8bca3a4824a